README.md

```diff
@@ -1,185 +1,268 @@
+```fortran
+  mkdir build
+  cd build
+  cmake -D EXAMPLES=ON -D MPI=ON -D BUILD_SHARED_LIBS=ON ..
+  make
+  make install
+```
 
-To use parpack from CMake, use PARPACK::PARPACK target:
+> Do not miss the `..` in line 3, as it denotes the location of source files.
 
-    find_package(arpack-ng)
-    add_executable(main main.f)
-    target_include_directories(main PUBLIC PARPACK::PARPACK)
-    target_link_libraries(main PARPACK::PARPACK)
+> The above script will builds everything including examples and parallel support (with MPI).
+
+### Using ARPACK-NG in Cmake project
+
+You can use arpack in your CMake builds by using `ARPACK::ARPACK` target. For example,
+
+```cmake
+  FIND_PACKAGE(arpack-ng)
+  ADD_EXECUTABLE(main main.f)
+  TARGET_INCLUDE_DIRECTORIES(main PUBLIC ARPACK::ARPACK)
+  TARGET_LINK_LIBRARIES(main ARPACK::ARPACK)
+```
+
+To use PARPACK in your Cmake builds, use `PARPACK::PARPACK` target:
+
+```cmake
+  FIND_PACKAGE(arpack-ng)
+  ADD_EXECUTABLE(main main.f)
+  TARGET_INCLUDE_DIRECTORIES(main PUBLIC PARPACK::PARPACK)
+  TARGET_LINK_LIBRARIES(main PARPACK::PARPACK)
+```
+
+### Extras
 
 On mac OS, with GNU compilers, you may need to customize options:
 
-    $ LIBS="-framework Accelerate" FFLAGS="-ff2c -fno-second-underscore" FCFLAGS="-ff2c -fno-second-underscore" ./configure
+```bash
+LIBS="-framework Accelerate" FFLAGS="-ff2c -fno-second-underscore" FCFLAGS="-ff2c -fno-second-underscore" ./configure
+```
 
 To build with code coverage:
-   
-    $ mkdir build
-    $ cd build
-    $ cmake -DCOVERALLS=ON -DCMAKE_BUILD_TYPE=Debug ..
-    $ make all check test coveralls
-
-To get iso_c_binding support:
-   
-    $ ./configure --enable-icb
-    $ cmake -D ICB=ON
-   The install will now provide arpack.h/hpp, parpack.h/hpp and friends.
-   Examples of use can be found in ./TESTS and ./PARPACK/TESTS/MPI.
-   A few related links can be found here:
-
-   * http://fortranwiki.org/fortran/show/ISO_C_BINDING
-   * http://fortranwiki.org/fortran/show/Generating+C+Interfaces
-   * https://www.roguewave.com/sites/rw/files/attachments/StandardizedMixedLanguageProgrammingforCandFortran.pdf
-
-5. Within DOCUMENTS directory there are three files
-
-* ex-sym.doc
-* ex-nonsym.doc and
-* ex-complex.doc
-
-   for templates on how to invoke the computational modes of ARPACK.
-   Also look in the README.MD file for explanations concerning the
-   other documents.
- 
-Custom installation examples:
-    
-    $ LIBSUFFIX="64" ./configure; make all install
-    $ cmake -DLIBSUFFIX="64" ..; make all install
-    
-    $ INTERFACE64="1" ITF64SUFFIX="ILP64" ./configure; make all install
-    $ cmake -DINTERFACE64=ON -DITF64SUFFIX="ILP64" ..; make all install
-    
-    $ LIBSUFFIX="64" INTERFACE64="1" ITF64SUFFIX="ILP64" ./configure; make all install
-    $ cmake -DLIBSUFFIX="64" -DINTERFACE64=ON -DITF64SUFFIX="ILP64" ..; make all install
+
+```bash
+mkdir build
+cd build
+cmake -DCOVERALLS=ON -DCMAKE_BUILD_TYPE=Debug ..
+make all check test coveralls
+```
```

Comment:
Coverage section can be killed: doesn't work anymore... @sylvestre: OK with that?